src/front-end/typescript/lib/pages/user/profile/tab/organizations.tsx

```diff
@@ -420,7 +421,16 @@ function affiliatedTableBodyRows(
       {
         children: (
           <div>
-            <span>{affiliation.organization.legalName}</span>
+            {memberIsAdmin(affiliation) ? (
```

Comment:
could consider more specific naming convention as you have done in the back-end to disambiguate from application Admins, with `memberIsOrgAdmin`

Reply:
good idea